extensions/wikia/PhalanxII/classes/Phalanx.class.php

```diff
@@ -288,7 +317,10 @@ private function log( $action ) {
 		$log->addEntry(
 			$action,
 			$title,
-			wfMsgExt( 'phalanx-rule-log-details', array( 'parsemag', 'content' ), $this->data['text'], $types, $this->data['reason'] )
+			wfMsgExt(
+				'phalanx-rule-log-details', [ 'parsemag', 'content' ],
+				$this->data['text'], $types, $this->data['reason']
+			)
```

Comment:
@garthwebb we should use `Message::inContentLanguage` for that, ie: `wfMessage(...)->inContentLanguage()->escaped();`